Libraries/LibJS/Runtime/Realm.h

```diff
@@ -71,6 +71,12 @@ class Realm final : public Cell {
         return *m_builtins[to_underlying(builtin)];
     }
 
+    // See: https://github.com/tc39/proposal-shadowrealm/pull/415
+    // This should be removed after the merge of https://github.com/tc39/ecma262/pull/3374 which applies the same fix
+    // as this slot in Realm achieves.
+    bool is_shadow_realm(Badge<VM>) const { return m_is_shadow_realm; }
```

Comment:
This is... not nice.. But I think it might be the best approach until ecma262#3374 is merged

Reply:
Oh dang, I didn't run the test-js tests against the Realm change. I'll investigate, may need to report that back to that merge request. Will leave it in draft for now.